commands/delete-key.js

```diff
@@ -20,6 +21,24 @@ async function deleteAccessKey(options) {
     console.log(`Deleting key = ${options.accessKey} on ${options.accountId}.`);
     const near = await connect(options);
     const account = await near.account(options.accountId);
-    const result = await account.deleteKey(options.accessKey);
-    inspectResponse.prettyPrintResponse(result, options);
+    let accessKeys = await account.getAccessKeys();
+    let fullAccessKeys = accessKeys.filter(accessKey => accessKey.access_key.permission === 'FullAccess');
+
+    if (fullAccessKeys.length === 1 && fullAccessKeys[0].public_key.includes(options.accessKey)){
+        const rl = readline.createInterface({
+            input: process.stdin,
+            output: process.stdout
+        });
+
+        const question = (str) => new Promise(resolve => rl.question(str, resolve));
+        const answer = await question('WARN: you want to remove the last full access key and forgot access to this account [Y/n]? ');
+
+        if (['YES', 'Yes', 'yes', 'Y', 'y'].includes(answer)) {
+            const result = await account.deleteKey(options.accessKey);
+            inspectResponse.prettyPrintResponse(result, options);
+        } else {
+            rl.close();
+            console.log('Deleting key canceled.');
+        }
+    }
```

Comment:
We actually have a ready-to-use utility [function](https://github.com/near/near-cli/blob/master/utils/readline.js#L3) for asking y/n questions; you can use [this example](https://github.com/near/near-cli/blob/master/utils/check-credentials.js#L17-L21) as a reference.

Reply:
Used `askYesNoQuestion`

Reply:
Seems to me now in brach `master` we have this version of code:
   >>>>>>> master
    
    if(fullAccessKeys.length === 1 && fullAccessKeys[0].public_key.includes(options.accessKey)){
        console.log('You can\'t delete last FullAccess key');
    } else {
        const result = await account.deleteKey(options.accessKey);
        inspectResponse.prettyPrintResponse(result, options);
    }
    
    @volovyk-s 
should we close this PR cause we really not needed to ask customers are they really want to remove last fullAccess key - seems to me we already have proper decision in mater ?

Reply:
I am not sure where you found this code, but I can't see it anywhere in `master`. Do you have a link?